File: app/src/main/java/com/task/paginationlist/repositories/RepoResult.java
package com.task.paginationlist.repositories;


public class RepoResult {

    public boolean wasDeleted = false;
    public boolean isUpdated = false;
    public long id;

    public RepoResult(boolean isUpdated, long id) {
        this.isUpdated = isUpdated;
        this.id = id;
    }

    public RepoResult(boolean wasDeleted) {
        this.wasDeleted = true;
    }

}

File: app/src/main/java/com/task/paginationlist/repositories/RepoResult.kt
package com.task.paginationlist.repositories


class RepoResult {

    var wasDeleted = false
    var isUpdated = false
    var id: Long = 0

    constructor(isUpdated: Boolean, id: Long) {
        this.isUpdated = isUpdated
        this.id = id
    }

    constructor(wasDeleted: Boolean) {
        this.wasDeleted = true
    }

}
